Source/qol/itemlabels.cpp

```diff
@@ -0,0 +1,171 @@
+/**
+* @file itemlabels.cpp
+*
+* Adds Item Labels QoL feature
+*/
+
+#include "common.h"
+#include "itemlabels.h"
+#include "inv.h"
+#include "render.h"
+#include "gmenu.h"
+#include "cursor.h"
+

```

Comment:
```suggestion

#include <cstring>
```

Reply:
?

Reply:
ah nvm, github made me think you wanted to replace render/gmenu/cursor includes with cstring lol